client/modules/IDE/components/About.jsx

```diff
@@ -67,7 +67,7 @@ function About(props) {
             rel="noopener noreferrer"
           >
             <AsteriskIcon className="about__content-column-asterisk" aria-hidden="true" focusable="false" />
-            {t('Reference')}
+            {t('Nav.Help_DropDown.Reference')}
```

Comment:
Again, I wonder if these could be shorter?

`Nav.Help_DropDown.Reference` vs `Nav.Help.Reference`?

I'm not sure though, what do you think?

Reply:
In this case the issue is that when I retrieve Nav.Help will get me an object and i18n will complain

I can switch to this convention, with Title as the label that will go in the DropDown 

"Help": {
  "Title": "Help",
  "KeyboardShortcuts": "Keyboard Shortcuts",
  "Reference": "Reference",
  "About": "About"
},
Notice that this convention will be required in the other dropdowns in Nav menu
what do you think?

Reply:
Yes, that’s much clearer I think. Let’s make that change everywhere if possible.